resources/user-agents/browsers/uc-browser/uc-web-10.json

```diff
@@ -222,19 +222,30 @@
         {
           "match": "Mozilla/5.0 (#PLATFORM# Build/*) applewebkit* (*khtml*like*gecko*) Version/* UCBrowser/#MAJORVER#.#MINORVER#* U3/* Safari/*",
           "platforms": [
-            "Android_2_3", "Android_4_0", "Android_4_1", "Android_4_2", "Android_4_3", "Android_4_4", "Android_5_0", "Android_5_1", "Android_6_0", "Android_6_1", "Android_7_0", "Android_7_1", "Android"
```

Comment:
Most of these changes to UC Browser files (except for 8 and 9) are just clean-up (re-ordering android versions, making sure all children use the `#DEVICE#` placeholder to make it easier to add future devices, etc...).